CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+<p align="center">
+  <picture>
+    <source
+      width="256px"
+      media="(prefers-color-scheme: dark)"
+      srcset="assets/revanced-headline/revanced-headline-vertical-dark.svg"
+    >
+    <img 
+      width="256px"
```

Comment:
Without `width="256"`, this caused the branding to be extra large for light mode users

PR opened @ https://github.com/ReVanced/revanced-branding/pull/29